scripts/build-static-libfesvr.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# This ungodly script surreptitiously builds an archive from existing fesvr objects
+# Invoke from riscv-fesvr/build
+
+if [ "x$RISCV" = "x" ]
+then
+  echo "Please set the RISCV environment variable to your preferred install path."
+  exit 1
+fi
+
+set -e
+
+objs=$(head -n 1 <(make -f <( echo -e 'include Makefile\n$(info $(value fesvr_objs))') -n))
```

Comment:
Wow this really is gross.

Reply:
This was just pulled from firesim into chipyard. I don't have any intention of touching this :)